scout/server/blueprints/variants/templates/variants/sv-variant.html

```diff
@@ -141,6 +141,7 @@
               {% else %}
                 - BAM file(s) missing
               {% endif %}
+              - <a class="btn btn-outline-secondary btn-sm" href="http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&position=chr{{ variant.chromosome }}:{{ variant.position }}&dgv=pack&knownGene=pack&omimGene=pack" target="_blank">UCSC</a>
```

Comment:
Looking good and the links work:

![image](https://user-images.githubusercontent.com/28093618/66653334-0e139980-ec38-11e9-9cb4-baa6954fb620.png)